src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -327,7 +326,7 @@ public String execute(Jedis connection) {
   }
 
   @Override
-  public Long expire(final byte[] key, final int seconds) {
+  public Long expire(final byte[] key, final long seconds) {
```

Comment:
(Not for this PR specifically so not suggesting you deal with it here, but just thining about next major release and if we should change that)

Expire can never return null [according to the docs](https://redis.io/commands/expire), do we know why we have some return values using the primitive wrappers and some not?

Reply:
@mina-asham My guesses:
- At the very core, Jedis uses `Object` (Connection.readProtocolWithCheckingBroken()). So that Object is being wrapped, `Long` came naturally.
- Long long ago, Redis wasn't clear and strict enough about respective responses. `Long` is a safer choice in such cases.

Reply:
> Redis wasn't clear and strict enough about respective responses

That's what I thought might be the case, I think we should strengthen the API and return `long` instead of `Long` where applicable, also relates to the below comment but per your request I will ignore that part and review the rest for now :)

Reply:
> return long instead of Long where applicable

@mina-asham I tried but it seems to be too complicated to include in this PR. IMHO, we should do this in separate PR. I have created #2543 to remind us.